.github/workflows/test-go-task.yml

```
@@ -197,6 +197,7 @@ jobs:
       - name: Send unit tests coverage to Codecov
         uses: codecov/codecov-action@v3
         with:
+          token: ${{ secrets.CODECOV_TOKEN }}
```

```suggestion
          token: ${{ vars.CODECOV_TOKEN }}
```

Reference:
https://docs.github.com/en/actions/learn-github-actions/variables#using-the-vars-context-to-access-configuration-variable-values